python/environment.yml

```
@@ -8,6 +8,15 @@ dependencies:
   # Code formatter and stuff
   - jedi-language-server
 
+  # Compilers
+  - libgcc-ng
+  - gcc_linux-64
+  - gxx_linux-64
+  - sysroot_linux-64
```

Need to get `sysroot_linux-64=2.28` or newer to fix the `ld: cannot find /usr/lib64/libpthread_nonshared.a inside /srv/conda/envs/notebook/bin/../x86_64-conda-linux-gnu/sysroot` issue.

```suggestion
  - sysroot_linux-64>=2.28
```